frontend/libs/studio-components/src/components/StudioActionCloseButton/StudioActionCloseButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef, type MouseEvent } from 'react';
+import { CheckmarkIcon } from '@studio/icons';
+import { StudioButton, type StudioButtonProps } from '../StudioButton';
+import type { OverridableComponent } from '../../types/OverridableComponent';
+
+export type StudioActionCloseButtonProps = StudioButtonProps & {
+  onClick: (event: MouseEvent<HTMLButtonElement>) => void;
+};
+
+const StudioActionCloseButton: OverridableComponent<
+  StudioActionCloseButtonProps,
+  HTMLButtonElement
+> = forwardRef<HTMLButtonElement, StudioActionCloseButtonProps>(
+  ({ onClick, variant = 'secondary', ...rest }: StudioActionCloseButtonProps, ref) => {
+    const handleOnClick = (event: MouseEvent<HTMLButtonElement>): void => {
+      onClick(event);
+    };
+
+    return (
+      <StudioButton
+        color='success'
+        icon={<CheckmarkIcon />}
+        onClick={handleOnClick}
+        variant={variant}
+        {...rest}
+        ref={ref}
+      />
+    );
+  },
+);
+
+StudioActionCloseButton.displayName = 'StudioActionCloseButton';
+
+export { StudioActionCloseButton };
```

Comment:
Why do we export it at the bottom and not on line 10? 😄 Are there any differences?

Reply:
No, should be fine to export it from the top.